tools/e2etesting/steps/cleanupaks.yml

```diff
@@ -0,0 +1,29 @@
+parameters:
+- name: CleanupAppRegistrations
+  type: boolean
+
+steps:
+- task: AzurePowerShell@5
+  displayName: "Delete Resource Group"
+  inputs:
+    azureSubscription: '$(AzureSubscription)'
+    azurePowerShellVersion: 'latestVersion'
+    scriptType: 'InlineScript'
+    inline: |
+      Write-Host "Deleting Resource group '$(ResourceGroupName)'..."
+      Get-AzResourceGroup -Name "$(ResourceGroupName)" | Remove-AzResourceGroup -Force
+
+# - task: AzurePowerShell@5
```

Comment:
commented out code can be deleted, we don't need App Registration, so we don't need to delete it